packages/playground/_stories/main/Button/Button.stories.ts

```diff
@@ -44,19 +44,17 @@ Basic.args = {
 	accessibleName: "Button with Accessible Name",
 };
 
-export const DifferentDesigns: StoryFn = () => html`
+export const DifferentTypes: StoryFn = () => html`
```

Comment:
This whole example seems redundant for me. You can easily achieve it purpose by changing the design property of the button inside Basic same. I think we can delete it

Reply:
In my opinion we should keep the `DifferentTypes` story, because you can achieve that by changing the `design` property, yes - but, by leaving it that way the user could easily see all the available designs at one glance and next to each other, instead of selecting the designs one by one.

Reply:
And what's the idea of having dynamical samples when they can't be edited dynamically?

Reply:
Sorry, Im not sure I understand the question, but I believe just simply to **showcase** different designs next to each other ?

Another good benefit in my opinion is that when user navigates through the `docs`, could also see the different designs, next to each other, since here samples are not interactable anyway:

![image](https://github.com/SAP/ui5-webcomponents/assets/88034608/1ca1c147-3b34-4725-845f-595605735e93)

Reply:
After internal discussions we've ended up allowing all those buttons to be editable dynamically in the sample.
The Icon-Only buttons are now only two (Transparent and Default) and they also have their properties editable dynamically.